devtools/index.js

```diff
@@ -1,4 +1,4 @@
-import { initDevTools } from './devtools';
-
-initDevTools();
-
+if (typeof process !== 'undefined' && process.env && process.env.NODE_ENV !== 'production') {
```

Comment:
Actually I think we might want to have this *on* by default when `process` is undefined - that'd be a browser context, which means someone is directly adding this to a page via a script tag.

```js
if (typeof process==='undefined' || process.env && process.env.NODE_ENV!=='production') {
  // ...
}
```